knowledge-base/actions/mikepenz/action-junit-report/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: "JUnit Report Action"
+github-token:
+  action-input:
+    input: github_token
```

Comment:
Please change input to `input: token` view [here](https://github.com/mikepenz/action-junit-report/blob/b6d6e9a8caa401e9cbe9c43f3349d267aed1b6f1/action.yml#L7)